config/lib/express.js

```diff
@@ -65,7 +65,7 @@ module.exports.initMiddleware = function (app) {
   }));
 
   // Initialize favicon middleware
-  app.use(favicon('./modules/core/client/img/brand/favicon.ico'));
+  app.use(favicon(app.locals.favicon));
```

Comment:
@ilanbiala `app.locals.favicon` is set to `modules/core/client/img/brand/favicon.ico`
So, does this still need to prepend `./`?